README.md

```diff
@@ -39,17 +8,16 @@
 2. Add `git+ssh://git@github.com/F-Secure/tests-selector.git#egg=tests_selector` as a dependency to `requirements.txt`
 3. Install dependencies
 4. Run initialization `tests_selector_init`
-5. Run affected tests `tests_selector` basing on uncommited changes
 
 ### Usage from source code
 
 #### Initialization
 
 1. Checkout the project
 2. In project directory run `make install` - that will:
-    * create virtual environment
-    * download all the dependencies
-    * install `tests_selector` as a tool
+   - create virtual environment
```

Comment:
@alexey-vyskubov is it still valid? I remember that tox is somehow related to that points.